packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -908,6 +908,11 @@ export const components = ({
 
         label: {
           padding: 0,
+
+          [`.${inputBaseClasses.root}.${inputBaseClasses.disabled} &`]: {
+            color: odysseyTokens.TypographyColorDisabled,
+            "-webkit-text-fill-color": odysseyTokens.TypographyColorDisabled
+          },
```

Comment:
We need to add types for all our styles using `-webkit`. I have 2 examples in there.

We actually need to add that type to all of them, but I'm fine if you only add it in this one place for now.